src/libs/ajax.js

```diff
@@ -690,13 +690,11 @@ export const Match = {
   findMatch: async (consentId, purposeId) => {
     const url = `${await Config.getApiUrl()}/match/${consentId}/${purposeId}`;
     const res = await fetchOk(url, Config.authOpts());
-    let answer = {};
     try {
-      answer = await res.json();
-    } catch (error) {
-      answer = {};
-    } finally {
+      const answer = await res.json();
```

Comment:
Not sure why the finally block was used here. Try and catch is sufficient to handle the response. Having a finally block will suspend try/catch operations until finally resolves, which can lead to unusual order of operations for async events if not planned properly. Syntax will now be flagged by linter.